src/components/Footer.tsx

```diff
@@ -106,7 +92,7 @@ const styles = {
             xs: 5,
             sm: 5,
         },
-        bgcolor: "#4dabf5",
+        bgcolor: "#4d4d4d",
```

Comment:
In this way, the footer colour will stay the same in both "light" and "dark" modes. I have the same issue with my code. We need to discuss how to set and use theme colours.